### DISCUSSION TITLE
How does GF work

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#344] [Q&A] [answered]
I know I've posted a lot this week, but I have a lot of questions about Codename. This time, I'm wondering how the GF thing works. I checked for the sprite, and it's a spritemap with two JSONs. Erm, what the... heck? 

A. WHY???????????
B. How does it work coding-wise I'm very intrigued?
![image](https://github.com/user-attachments/assets/f97204a5-8b21-42c2-95f0-73e23b680371)

**Comment by moxie-coder** [comment, marked as answer]
her sprites use an Atlas because it's more optimized and allows you to do more then a spritesheet, and they use a library called FlxAnimate to load atlases because Smokeys atlas library was awful and only supported 2018 atlas and wasn't as optimized

**Reply by THEkatinamicrowave** [reply]
thanks lol I was really confused :D
